ui/vty-lib/src/Ariadne/UI/Vty/Widget/Form/List.hs

```diff
@@ -46,7 +46,7 @@ drawListWidget focused ListWidgetState{..} = do
   return $
     B.clickable widgetName $
     B.vBox $
-    map drawItem $ zip [0..] items
+    zipWith (curry drawItem) [0..] items
```

Comment:
Much understandable wow.

Reply:
I agree that it is less transparent, but my own impression of simplicity does not always match that of others, so I wanted to see some opinions. I will revert this.